salt/modules/file.py

```diff
@@ -3341,13 +3341,6 @@ def get_managed(
             source_sum = __salt__['cp.hash_file'](source, saltenv)
             if not source_sum:
                 return '', {}, 'Source file {0} not found'.format(source)
-        # if its a local file
-        elif urlparsed_source.scheme == 'file':
-            file_sum = get_hash(urlparsed_source.path, form='sha256')
-            source_sum = {'hsum': file_sum, 'hash_type': 'sha256'}
-        elif source.startswith('/'):
-            file_sum = get_hash(source, form='sha256')
-            source_sum = {'hsum': file_sum, 'hash_type': 'sha256'}
         elif source_hash:
             protos = ('salt', 'http', 'https', 'ftp', 'swift', 's3')
```

Comment:
@terminalmage, can we add `'file'` to this list of accepted source_hash protos? Will that allow us to specify a hash in the form: `source_hash: file:///path/to/hashfile.hash`?